### PR TITLE
Yautja Weapon Gimmicks

### PR DESCRIPTION
**Pull request by realforest2001**
# About the pull request
Work on giving Yautja Weapons gimmicks again.

# Explain why it's good for the game
Work in Progress. Allows Yautja weapons to have some flair beyond being variably effective sticks.
# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
add: Yautja weapons now collect blood 'charges' for using their abilites.
add: Combistick now requires two hits to throw instead of one, but can store up to two throws.
add: Scythe now can perform an AOE attack similar to Ravager Eviscerate, after five successful hits.
balance: Reduced the probability for combistick blocking from 2/3 hits to 1/3
balance: Added a 33% chance that instead of knocking yautja mask off, it degrades the hug resistance. Also repaired the 15% hardcap on knock chance.
/:cl:

### CODE REVIEW

**Comment by realforest2001** [comment]
Pending review/discussion on the gimmick of bleeding as it doesn't really have much of an effect beyond the natural bleed

File: code/modules/cm_preds/yaut_weapons.dm [hunk 1]
```diff
@@ -140,6 +154,44 @@
...
+
+	if(ability_charge < ability_charge_max)
+		ability_charge += ability_charge_rate
+
+/obj/item/weapon/yautja/unique_action(mob/user)
+	if(user.get_active_hand() != src)
+		return FALSE
+	if(ability_charge < ability_cost)
+		to_chat(user, SPAN_WARNING("The blood reservoir is not full enough to do this!"))
+		return FALSE
```
Comment (Zonespace27) [inline code/modules/cm_preds/yaut_weapons.dm:194]:
```suggestion
		return FALSE
	return TRUE
```

File: code/modules/cm_preds/yaut_weapons.dm [hunk 2]
```diff
@@ -208,19 +251,58 @@
...
+	..()
+	ability_primed = !ability_primed
+	var/message = "You tighten your grip on [src], preparing to whirl it in a spin."
+	if(!ability_primed)
+		message = "You relax your grip on [src]."
+	to_chat(user, SPAN_WARNING(message))
 
-	return
+/obj/item/weapon/yautja/scythe/unique_action(mob/user)
+	..()
```
Comment (Zonespace27) [inline code/modules/cm_preds/yaut_weapons.dm]:
```suggestion
	. = ..()
	if(!.)
		return FALSE
```

**Review by Zonespace27 — COMMENTED** [review]
here's the fix

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Review by harryob — CHANGES_REQUESTED** [review]

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Pull request closed without merging**